AmazonChimeSDK/AmazonChimeSDKTests/internal/audio/DefaultAudioClientControllerTests.swift

```diff
@@ -70,7 +70,7 @@ class DefaultAudioClientControllerTests: CommonTestCase {
                                                                     attendeeId: attendeeId,
                                                                     joinToken: joinToken,
                                                                     callKitEnabled: callKitEnabled,
-                                                                    audioMode: .mono))
+                                                                    audioMode: .stereo48K))
```

Comment:
wat's the purpose of this change instead of new test case?

Reply:
The default is changed to `stereo48K` hence the test had to be updated